engine/src/execution_graph/logic_controllers/BatchAggregationProcessing.h

```diff
@@ -85,7 +113,7 @@ class ComputeAggregateKernel : public kernel {
                 }
 
                 this->add_to_output_cache(std::move(output));
-                batch_count++;
+                
```

Comment:
how are you tryacking the number of batches processed by the kernel without this?

Reply:
This moved to line 55